DBQueries.py

```diff
@@ -234,5 +237,21 @@ def select_emails_to_notify(self, users_id_list):
             logger.debug(f'Query is: {query}.')
             cur.execute(query, (users_id_list,))
             records = cur.fetchall()
-            logger.info(f"{cur.rowcount} rows affected.")
+            logger.info(f"{cur.rowcount} rows fetched.")
+        return records
+
+    def select_email_item_title_to_out_of_stock(self, items_id_list):
```

Comment:
i dont understand this function name.. rewrite it

Reply:
ohh thats cool, I agree 100% and forgot to change it!